src/main/java/com/hackclub/hccore/commands/TutorialCommand.java

```diff
@@ -0,0 +1,33 @@
+package com.hackclub.hccore.commands;
+
+import com.hackclub.hccore.HCCorePlugin;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class TutorialCommand implements CommandExecutor {
+    private static final String[] message = {ChatColor.RED + "Greetings! Welcome to the Hack Club vanilla Minecraft server!", "You can use /nick to set your nick name and /color to set your chat and name colors", ChatColor.RED + "Rules:\n", "Be nice.",  "No griefing.", "Don't build within 1000 blocks of spawn.", "If you want to contribute to plugin development head on over to the GitHub: " + ChatColor.RED + "https://github.com/hackclub/HCCore/. ", "Type /tutorial to see this at any time."};
```

Comment:
I suggest that you include the "greeting" part in a new player event so "greeting" does not message when you run the command again.